src/doc/src/reference/unstable.md

```diff
@@ -1243,9 +1243,9 @@ Keys that are supported:
 
 Example:
 ```toml
-# [PROGECT_DIR]/Cargo.toml
+# [PROJECT_DIR]/Cargo.toml
```

Comment:
For context: I noticed this typo and the thought was to go ahead and look at addressing other typos